samples/javascript_es6/01.browser-echo/package.json

```diff
@@ -1,47 +1,42 @@
 {
     "name": "browser-echo",
-    "version": "1.0.0",
+    "version": "1.1.0",
     "description": "BotBuilder v4 echo bot using botframework-webchat",
     "author": "Microsoft",
     "license": "MIT",
     "main": "app.js",
     "scripts": {
-        "build": "webpack",
+        "build": "webpack --mode=production",
         "start": "webpack-dev-server",
         "lint": "eslint .",
         "test": "echo \"Error: no test specified\" && exit 1"
     },
     "dependencies": {
-        "@babel/runtime": "^7.1.5",
-        "botbuilder-core": "^4.2.0",
-        "botframework-directlinejs": "~0.11.2",
-        "botframework-webchat": "4.2.0",
-        "core-js": "^3.0.1"
+        "botbuilder-core": "^4.5.0",
```

Comment:
Can you use `~` instead of `^`? We test the versions by minor, so 4.2 might not necessarily work with 4.5.